code/script.js

```diff
@@ -1,325 +1,1505 @@
+    img: "images/RiotX_ChampionList_wukong.jpg",
+    hair: "brown",
+    homeRegion: "ionia",
+    mainRole: "fighter",
+    accessories: ["facialhair"],
+  },
+  {
+    name: "Xayah",
+    img: "images/RiotX_ChampionList_xayah.jpg",
+    hair: "purple",
+    homeRegion: "ionia",
+    mainRole: "marksman",
+    accessories: ["headpiece", "weapon"],
+  },
+  {
+    name: "Xerath",
+    img: "images/RiotX_ChampionList_xerath.jpg",
+    hair: "none",
+    homeRegion: "shurima",
+    mainRole: "mage",
+    accessories: [],
+  },
+  {
+    name: "Xin Zhao",
+    img: "images/RiotX_ChampionList_xinzhao.jpg",
+    hair: "black",
+    homeRegion: "demacia",
+    mainRole: "fighter",
+    accessories: ["weapon"],
+  },
+  {
+    name: "Yasuo",
+    img: "images/RiotX_ChampionList_yasuo.jpg",
+    hair: "brown",
+    homeRegion: "ionia",
+    mainRole: "fighter",
+    accessories: ["weapon"],
+  },
+  {
+    name: "Yone",
+    img: "images/RiotX_ChampionList_yone.jpg",
+    hair: "black",
+    homeRegion: "ionia",
+    mainRole: "assassin",
+    accessories: ["weapon", "headpiece"],
+  },
+  {
+    name: "Yorick",
+    img: "images/RiotX_ChampionList_yorick.jpg",
+    hair: "black",
+    homeRegion: "shadowisles",
+    mainRole: "fighter",
+    accessories: ["weapon", "headpiece", "facialhair"],
+  },
+  {
+    name: "Yuumi",
+    img: "images/RiotX_ChampionList_yuumi.jpg",
+    hair: "white",
+    homeRegion: "bandlecity",
+    mainRole: "support",
+    accessories: ["headpiece"],
+  },
+  {
+    name: "Zac",
+    img: "images/RiotX_ChampionList_zac.jpg",
+    hair: "none",
+    homeRegion: "zaun",
+    mainRole: "tank",
+    accessories: [],
+  },
+  {
+    name: "Zed",
+    img: "images/RiotX_ChampionList_zed.jpg",
+    hair: "none",
+    homeRegion: "ionia",
+    mainRole: "assassin",
+    accessories: ["weapon", "headpiece"],
+  },
+  {
+    name: "Ziggs",
+    img: "images/RiotX_ChampionList_ziggs.jpg",
+    hair: "brown",
+    homeRegion: "zaun",
+    mainRole: "mage",
+    accessories: ["weapon", "headpiece"],
+  },
+  {
+    name: "Zilean",
+    img: "images/RiotX_ChampionList_zilean.jpg",
+    hair: "blue",
+    homeRegion: "none",
+    mainRole: "support",
+    accessories: [],
+  },
+  {
+    name: "Zoe",
+    img: "images/RiotX_ChampionList_zoe.jpg",
+    hair: "orange",
+    homeRegion: "targon",
+    mainRole: "mage",
+    accessories: [],
+  },
+  {
+    name: "Zyra",
+    img: "images/RiotX_ChampionList_zyra.jpg",
+    hair: "red",
+    homeRegion: "ixtal",
+    mainRole: "mage",
+    accessories: [],
+  },
+];
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret,
+  currentQuestion,
+  charactersInPlay,
+  championToCheck,
+  startTime,
+  endTime;
+let questionsAsked = 0;
 
-// Draw the game board
+// Function that draws the game board
 const generateBoard = () => {
-  board.innerHTML = ''
-  charactersInPlay.forEach((person) => {
+  board.innerHTML = ``;
+  charactersInPlay.forEach((champion) => {
```

Comment:
You have even named this variables to fit your theme of the game (champions instead of "person") well done!